### PR TITLE
feat: Not Found Page added for fallback

### PR DESCRIPTION
**Pull request by Udit-takkar**
## Summary
Added a 404 Not Found Page.

Resolves isuue: #9 

Screenshot 2021-12-11 at 3 21 01 PM

### CODE REVIEW

**Comment by Udit-takkar** [comment]
@YashKumarVerma  Can you review?

**Review by sohamsshah — APPROVED** [review]
LGTM! 🎉

File: src/App.js [hunk 1]
```diff
@@ -6,19 +6,21 @@ import { Navbar } from "./components/navbar";
 /** import all pages into application */
 import { HomePage } from "./pages/homepage";
 import { DashboardPage } from "./pages/dashboard";
+import NotFoundPage from "./pages/NotFound";
```
Comment (YashKumarVerma) [inline src/App.js]: please do an object export like done for DashboardPage

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (Udit-takkar) [reply src/App.js]: Okay

**Pull request closed without merging**